cli/pkg/config_manager/k8s.go

```diff
@@ -0,0 +1,52 @@
+package config_manager
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	backend "github.com/chanzuckerberg/happy/shared/backend/aws"
+	"github.com/chanzuckerberg/happy/shared/config"
+	"github.com/chanzuckerberg/happy/shared/k8s"
+	"github.com/chanzuckerberg/happy/shared/util"
+	"github.com/pkg/errors"
+)
+
+func listEKSClusterIDs(ctx context.Context, profile, region string) ([]string, error) {
+	b, err := backend.NewAWSBackend(ctx, config.EnvironmentContext{
+		AWSProfile:     util.String(profile),
+		AWSRegion:      util.String(region),
+		TaskLaunchType: util.LaunchTypeNull,
+	})
+	if err != nil {
+		return []string{}, errors.Wrap(err, "unable to create an aws backend")
+	}
+	return b.ListEKSClusterIds(ctx)
+}
+
+func listHappyNamespaces(ctx context.Context, profile, region, clusterId string) ([]string, error) {
+	b, err := backend.NewAWSBackend(ctx, config.EnvironmentContext{
```

Comment:
you can call NewK8SComputeBackend or type inter the interface to the K8S backend struct like

~~~
k8sBackend := b.(K8sComputeBackend)
~~~
. Please don't add a K8s method the generic compute interface. It doesn't make sense

Reply:
That won't work, but I have a workaround.